source/installation/install-kafka-cluster.txt

```diff
@@ -0,0 +1,146 @@
+.. _install-kafka-cluster:
+
+======================================
+Install with an Existing Kafka Cluster
+======================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator on an existing Kafka cluster including 
+Kafka Connect. This installation method is ideal if you are already running 
+Kafka or you have your own approach to installing and configuring Kafka.
+
+Before you Begin 
+----------------
+
+You must have Kafka and a Kafka Connect cluster accessible to the computer 
+running Relational Migrator.
```

Comment:
**Suggestion**

Slight tweak to clarify what accessibility means in this lead-in sentence:

```suggestion
You must have Kafka and network connectivity to the Kafka Connect cluster from the computer 
running Relational Migrator.
```

Reply:
Updated.